package.json

```diff
@@ -9,9 +9,9 @@
     "fetchCoverage": "./node_modules/.bin/grunt exec:fetchCoverage --verbose",
     "coverageReport": "./node_modules/.bin/grunt coverageReport --verbose",
     "test": "./node_modules/.bin/grunt test --verbose",
-    "test-unit": "./node_modules/.bin/grunt test-unit --verbose",
-    "test-integration": "export TEST=test/integration/peers.integration.js && export NODE_ENV=TEST && ./node_modules/.bin/grunt jenkins",
-    "test-functional": "grunt test-functional",
+    "test-unit": "./node_modules/.bin/grunt testUnit --verbose",
+    "test-functional": "./node_modules/.bin/grunt testFunctional --verbose",
+    "test-integration": "./node_modules/.bin/grunt testIntegration",
```

Comment:
Because it is a small change for development purposes and the discussion started thanks to this PR. @Isabello do really you want me to remove these changes and open a new issue/PR ?